scripts/release/create-release.hs

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env cabal 
+{- project:
+  with-compiler: ghc-9.4
```

Comment:
I think (I at least) we use 8.10 officially, 9.2 in every day's work and we aim to support 9.6 in the future. Why did you choose 9.4?